core/src/test/scala/filodb.core/memstore/TimeSeriesMemStoreForMetadataSpec.scala

```diff
@@ -56,15 +56,14 @@ class TimeSeriesMemStoreForMetadataSpec extends FunSpec with Matchers with Scala
     val filters = Seq (ColumnFilter("__name__", Filter.Equals("http_req_total".utf8)),
       ColumnFilter("job", Filter.Equals("myCoolService".utf8)))
     val metadata = memStore.partKeysWithFilters(timeseriesDataset.ref, 0, filters, now, now - 5000, 10)
-    val schema = new RecordSchema(Seq(ColumnType.PartitionKeyColumn))
+    val schema = new RecordSchema(Seq(ColumnType.MapColumn))
```

Comment:
Why change this test?  These changes have nothing to do with `indexValuesWithFilters` which you changed.